system-int-test/test/cmr/system_int_test/ingest/service_ingest_test.clj

```diff
@@ -257,3 +257,77 @@
+                                  "Format" "invalid"
+                                  "MimeType" "invalid"}]
+          concept-src1 (service-util/make-service-concept)
+          concept-result1 (as-> concept-src1 intermediate
+                                (json/parse-string (:metadata intermediate))
+                                (assoc intermediate "RelatedURLs" related-urls-valid)
+                                (json/generate-string intermediate)
+                                (assoc concept-src1 :metadata intermediate)
+                                (ingest/ingest-concept intermediate))
+          concept-src2 (service-util/make-service-concept)
+          concept-result2 (as-> concept-src2 intermediate
+                                (json/parse-string (:metadata intermediate))
+                                (assoc intermediate "RelatedURLs" related-urls-invalid1)
+                                (json/generate-string intermediate)
+                                (assoc concept-src2 :metadata intermediate)
+                                (ingest/ingest-concept intermediate))
+          concept-src3 (service-util/make-service-concept)
+          concept-result3 (as-> concept-src3 intermediate
+                                (json/parse-string (:metadata intermediate))
+                                (assoc intermediate "RelatedURLs" related-urls-invalid2)
+                                (json/generate-string intermediate)
+                                (assoc concept-src3 :metadata intermediate)
+                                (ingest/ingest-concept intermediate))
+          concept-src4 (service-util/make-service-concept)
+          concept-result4 (as-> concept-src4 intermediate
+                                (json/parse-string (:metadata intermediate))
+                                (assoc intermediate "RelatedURLs" related-urls-invalid3)
+                                (json/generate-string intermediate)
+                                (assoc concept-src4 :metadata intermediate)
+                                (ingest/ingest-concept intermediate))
+          status1 (:status concept-result1)
+          errors1 (:errors concept-result1)
+          status2 (:status concept-result2)
+          errors2 (:errors concept-result2)
+          status3 (:status concept-result3)
+          errors3 (:errors concept-result3)
+          status4 (:status concept-result4)
+          errors4 (:errors concept-result4)]
+      (is (= 201 status1))
+      (is (= 0 (count errors1)))
+      (is (= 400 status2))
+      (is (= 1 (count errors2)))
+      (is (= [{:path ["RelatedUrLs" 0 "Format"], :errors ["Format [invalid] was not a valid keyword."]}] errors2))
+      (is (= 400 status3))
+      (is (= 1 (count errors3)))
+      (is (= [{:path ["RelatedUrLs" 0 "MimeType"], :errors ["MimeType [invalid] was not a valid keyword."]}] errors3))
+      (is (= 400 status4))
+      (is (= 2 (count errors4)))
```

Comment:
Could these be grouped into `testing` blocks or comments added to show what is being tested more clearly?
And is `RelatedUrLs` a typo? should the `r` be upper-case as well?

Reply:
I could add some comments.  RelatedUrLs is NOT a typo. All the existing tests have it spelled like that and it's what the validation error contains.  Not sure where it got that spelling from.